test/test_env.py

```diff
@@ -992,6 +992,32 @@ def test_steptensordict(
         assert out is next_tensordict
 
 
+@pytest.mark.skipif(not _has_gym, reason="no gym")
+@pytest.mark.parametrize("env_name", ["Pendulum-v1", "ALE/Pong-v5"])
+@pytest.mark.parametrize("frame_skip", [1, 4])
+def test_batch_lock_gym(env_name, frame_skip, seed=0):
+    env = GymEnv(env_name, frame_skip=frame_skip)
+    assert env.batch_locked
+
+    try:
```

Comment:
you can do 
```
with pytest.mark.raise
```

There are a couple of examples in the tests if you'd like to have a look